.appveyor.yml

```diff
@@ -18,12 +18,14 @@ branches:
   - try
 
 install:
+# Setting build env
+- set HOST_TARGET=%HOST_TARGET%
```

Comment:
Isn't this redundant? This seems to set a variable to itself...

Reply:
You're right. Skimming through their docs says something like that: <https://www.appveyor.com/docs/build-configuration/#environment-variables>.